benchmarks/different_model_options.py

```diff
@@ -33,8 +33,8 @@ def build_model(parameter, model_, option, value):
 class SolveModel:
     solver: pybamm.BaseSolver
     model: pybamm.BaseModel
-    t_eval: np.ndarray
-    t_interp: np.ndarray | None
+    t_eval: numpy.typing.NDArray
```

Comment:
We are importing `numpy` as `np` so this would obviously not work. In fact, we should `import numpy.typing as npt` and use `npt.NDArray` everywhere.